docs/Debugging.md

```diff
@@ -71,6 +73,20 @@ The debugger will receive a list of all project roots, separated by a space. For
 
 > Custom debugger commands executed this way should be short-lived processes, and they shouldn't produce more than 200 kilobytes of output.
 
+## React Developer Tools
+
+With React Native 0.43 or higher, you can use [the standalone version of React Developer Tools](https://github.com/facebook/react-devtools/tree/master/packages/react-devtools) to debug the React component hierarchy. To use it, install the `react-devtools` package globally:
+
+```js
+npm install -g react-devtools
+```
+
+Now run `react-devtools` from the terminal to launch the standalone DevTools app.
+
+It should connect to your simulator within a few seconds.
+
+> Note: if you prefer to avoid global installations, alternatively you can run `npm install --save-dev react-devtools` and add `"react-devtools": "react-devtools"` to the `scripts` section in your `package.json`. Then you could run `npm run react-devtools` from your project folder.
```

Comment:
Not sure if you care but with Yarn you can run any script in `node_modules/.bin` by just running `yarn <script-name>`. So for example with Yarn all you need to do is `yarn add react-devtools; yarn react-devtools`.